python/cudf/cudf/core/series.py

```diff
@@ -2597,6 +2597,26 @@ def is_monotonic_decreasing(self):
     def __cuda_array_interface__(self):
         return self._column.__cuda_array_interface__
 
+    def repeat(self, repeats, axis=None):
+        assert axis in (None, 0)
+        in_index = self.index
+        from cudf.core.multiindex import MultiIndex
+
+        if isinstance(in_index, MultiIndex):
+            in_index = RangeIndex(len(in_index))
```

Comment:
This should actually repeat the values in the MultiIndex instead of changing it to a RangeIndex and repeating that.